astropy/io/ascii/ui.py

```diff
@@ -175,9 +175,18 @@ def read(table, guess=None, **kwargs):
         List of names to include in fill_values (default= ``None`` selects all names)
     fill_exclude_names : list
         List of names to exclude from fill_values (applied after ``fill_include_names``)
-    fast_reader : bool
+    fast_reader : bool or dict
         Whether to use the C engine, can also be a dict with options which default to ``False``
-        (default= ``True``)
+        (default= ``True``); parameters for options dict:
+
+        use_fast_converter: bool
```

Comment:
I don't think the nested argument formatting like this works.  I think you can just use a list of dictionary keys here, i.e.:

```
- ``use_fast_converter`` (bool): enable...
- ``parallel`` (bool or int): multiprocessing...
```